PelotonEppSdk/Models/EventRequest.cs

```diff
@@ -2,75 +2,100 @@
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Net;
 using System.Threading.Tasks;
+using System.Web;
 using PelotonEppSdk.Classes;
 using PelotonEppSdk.Enums;
 
 namespace PelotonEppSdk.Models
 {
     public class EventRequest : RequestBase
     {
-        [Required]
         [StringLength(32, MinimumLength = 32, ErrorMessage = nameof(EventToken) + " must be 32 characters long.")]
         public string EventToken { get; set; }
 
+        /// <summary>
+        /// ClientToken for the target peloton account
+        /// </summary>
+        //[Required]
```

Comment:
nit: commented out require annotations are being added. Should these be removed?